pkg/apis/externaldns/types.go

```diff
@@ -503,6 +509,9 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("infoblox-name-regex", "Apply this regular expression as a filter on the name field for obtaining infoblox records. This is disabled by default.").Default(defaultConfig.InfobloxNameRegEx).StringVar(&cfg.InfobloxNameRegEx)
 	app.Flag("infoblox-create-ptr", "When using the Infoblox provider, create a ptr entry in addition to an entry").Default(strconv.FormatBool(defaultConfig.InfobloxCreatePTR)).BoolVar(&cfg.InfobloxCreatePTR)
 	app.Flag("infoblox-cache-duration", "When using the Infoblox provider, set the record TTL (0s to disable).").Default(strconv.Itoa(defaultConfig.InfobloxCacheDuration)).IntVar(&cfg.InfobloxCacheDuration)
+	app.Flag("infoblox-tenant-id", "Add Tenant Id as an extensible attribute. This is disabled by default.").Default(defaultConfig.InfobloxTenantId).StringVar(&cfg.InfobloxTenantId)
```

Comment:
```suggestion
	app.Flag("infoblox-tenant-id", "When using the Infoblox provider, add Tenant Id as an extensible attribute. This is disabled by default.").Default(defaultConfig.InfobloxTenantId).StringVar(&cfg.InfobloxTenantId)
```

and so on.

Reply:
I've updated all three flag descriptions in the MR